sentry_sdk/_lru_cache.py

```diff
@@ -0,0 +1,156 @@
+"""
```

Comment:
Not sure if this will suffice but it's copied from https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/_queue.py